dashboards/victoriametrics.json

```diff
@@ -698,12 +698,12 @@
       "renderer": "flot",
       "seriesOverrides": [],
       "spaceLength": 10,
-      "stack": false,
+      "stack": true,
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum(rate(vm_rows_inserted_total{job=~\"$job\", instance=~\"$instance\"}[5m])) by (type) > 0 ",
-          "legendFormat": "{{type}}",
+          "expr": "sum(rate(vm_rows_inserted_total{job=~\"$job\", instance=~\"$instance\"}[5m])) by (type,accountID) > 0 ",
+          "legendFormat": "{{type}} {{accountID}}",
```

Comment:
Would it make sense to drop the format, so Grafana could name displayed values and print this:
```
{accountID="0", type="promremotewrite"}
```
Instead of this:
```

promremotewrite 0
```
wdyt?

Reply:
I have no special idea behind that. Removing `legendFormat` looks legit for me too.

Reply:
Let's do that then. Could you pls update the PR?